win32ss/reactx/ntddraw/mocomp.c

```diff
@@ -63,11 +64,11 @@ NtGdiDdDestroyMoComp(HANDLE hMoComp,
 
     if (pfnDxDdDestroyMoComp == NULL)
     {
-        DPRINT1("Warning: no pfnDxDdDestroyMoComp");
+        DPRINT1("Warning: no pfnDxDdDestroyMoComp\n");
```

Comment:
Make this function's alias same as others (without `Dx` prefix, since we already have `pfn`):
```suggestion
        DPRINT1("Warning: no pfnDdDestroyMoComp\n");
```